benchmarks/README.md

```diff
@@ -0,0 +1,15 @@
+# Benchmarks, to be run by codspeed.io in CI
+
+## Running benchmarks
+
+Note that as of mid-2024, codspeed.io uses a simulated CPU (Cachegrind) to run tests, so the measures of performance are not suitable for optimizing low-level compiled code.
+
+In CI, we use `tox -e benchmarks-ci` to run the benchmarks.
```

Comment:
Maybe add a note that normal GitHub Action VMs are used for CI ... there are no dedicated servers.

```suggestion
In CI, we use `tox -e benchmarks-ci` to run the benchmarks, using GitHub Actions.
The tests are executed on GitHub VMs and the reports are sent to the codspeed.io cloud.
```